src/Api/Response/Response.php

```diff
@@ -172,6 +163,54 @@ public function getQueryInfoMessage(): string
         }
     }
 
+    public function getSmartDidYouMean(): string
+    {
+        $dataQueryInfoMessage = $this->getData(self::DATA_QUERY_INFO_MESSAGE);
+
+        if (empty($dataQueryInfoMessage)) {
+            return '';
+        }
+
+        $type = $dataQueryInfoMessage['queryStringType'];
+        $alternativeQuery = $dataQueryInfoMessage['currentQuery'];
+        $originalQuery = $dataQueryInfoMessage['originalQuery'];
+
+        if (!empty($dataQueryInfoMessage['didYouMeanQuery'])) {
+            $type = 'did-you-mean';
+            $alternativeQuery = $dataQueryInfoMessage['didYouMeanQuery'];
```

Comment:
Isn't this the same as in line 116? Can this be combined somehow? If not, create a constant for the string 'did-you-mean' and use it on both places, otherwise we may forgot to update it in one place in the future.